src/wallet/rpc/wallet.cpp

```diff
@@ -220,7 +220,7 @@ static RPCHelpMan loadwallet()
                 "\nNote that all wallet command-line options used when starting bitcoind will be"
                 "\napplied to the new wallet.\n",
                 {
-                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The wallet directory or .dat file."},
+                    {"filename", RPCArg::Type::STR, RPCArg::Optional::NO, "The path to the wallet directory (absolute or relative to the wallets directory), or a legacy wallet's .dat file (within the wallets directory). The default wallets directory is the 'wallets' folder inside the data directory, unless the `-walletdir` option is specified."},
```

Comment:
"The default wallets directory is the 'wallets' folder inside the data directory, unless the `-walletdir` option is specified."

This sentence is a little bit confusing as it suggests `-walletdir` changes the default somehow. I would rephrase this as "The wallets directory is set by the `-walletdir` option and defaults to the 'wallets' folder within the data directory."